x-pack/plugins/monitoring/public/views/apm/instance/index.html

```diff
@@ -0,0 +1,8 @@
+<monitoring-main
+  product="apm"
+  name="apm"
+  instance="{{ pageData.apmSummary.name }}"
+  data-test-subj="apmInstancePage"
+>
+  <monitoring-apm-server-instance data="pageData"></monitoring-apm-server-instance>
```

Comment:
We're able to reduce the amount of Angular/React mix by putting a DOM element here for React to mount to, and have the view render react into this template. I've added a helper view controller class called `MonitoringViewBaseController`. That should help reduce a lot of the boilerplate bulk. IIRC, you don't even need a `getPageData` function.

`public/views/elasticsearch/overview/controller.js` is an example

Reply:
Sounds good!